<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>originalProcess</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_fork">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>fork</string>
    </property>
   </widget>
   <widget class="QLabel" name="lable_procInfo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_childProcName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>264</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
